### PR TITLE
PLAT-124521: Fix the text size in ArcPicker is smaller than the one in ArcSlider

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - added `font-size` and `font-weight` to `ArcPicker` component to share the same styles as `ArcSlider`
 - removed `font-size` style from `FanSpeedControl` as now it inherits it from `ArcPicker`

### Links
[//]: # (Related issues, references)
PLAT-124521

### Comments

Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by ybsung — APPROVED** [review]
LGTM

File: ArcPicker/ArcPicker.module.less [hunk 1]
```diff
@@ -23,6 +23,8 @@
 	.applySkins({
 		.valueDisplay {
 			height: @agate-arc-picker-value-display-height;
+			font-size: @agate-arcpicker-font-size;
+			font-weight: @agate-arcpicker-font-weight;
```
Comment (MikyungKim) [inline ArcPicker/ArcPicker.module.less:27]:
```suggestion
			font-size: @agate-arc-picker-font-size;
			font-weight: @agate-arc-picker-font-weight;
```

File: styles/variables-gallium.less [hunk 2]
```diff
@@ -4,6 +4,8 @@
 // ArcPicker
 // ---------------------------------------
 @agate-arc-picker-value-display-height: 120px;
+@agate-arcpicker-font-size: 72px;
+@agate-arcpicker-font-weight: 300;
```
Comment (MikyungKim) [inline styles/variables-gallium.less:8]:
```suggestion
@agate-arc-picker-font-size: 72px;
@agate-arc-picker-font-weight: 300;
```

File: styles/variables-silicon.less [hunk 3]
```diff
@@ -27,6 +27,8 @@
 // ArcPicker
 // ---------------------------------------
 @agate-arc-picker-value-display-height: 120px;
+@agate-arcpicker-font-size: 72px;
+@agate-arcpicker-font-weight: 300;
```
Comment (MikyungKim) [inline styles/variables-silicon.less:31]:
```suggestion
@agate-arc-picker-font-size: 72px;
@agate-arc-picker-font-weight: 300;
```

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**